examples/Admin/policies.py

```diff
@@ -78,19 +78,18 @@ def bulk_delete_section(policy_keys, print_response=False):
         pretty = json.dumps(response, indent=4, sort_keys=True, default=str)
         print(pretty)
 
-def update_policy_with_device_health_app(policy_key, print_response=False):
+def update_policy_with_duo_desktop(policy_key, print_response=False):
     """
-    Update a given policy to include Duo Device Health App policy
+    Update a given policy to include Duo Desktop policy
     settings. Requires Access or Beyond editions.
     """
 
     json_request = {
         "sections": {
-            "device_health_app": {
+            "duo_desktop": {
```

Comment:
So I gather this has been updated on the Duo side?  if old clients are still around and sending the old property, is Duo just ignoring it?

Reply:
Right now Duo is able to still able to handle the old property, but the new property has been around since November 2023.